chapter3/user-memory/test_evaluation_memory_view.py

```
@@ -0,0 +1,63 @@
+        def __init__(self):
+            self.test_suite = FakeTestSuite()
+
+    fake_evaluation_modules = {
+        "config": types.SimpleNamespace(),
+        "models": types.SimpleNamespace(),
+        "evaluator": types.SimpleNamespace(),
+        "framework": types.SimpleNamespace(UserMemoryEvaluationFramework=FakeFramework),
+    }
+    real_import = builtins.__import__
+
+    def import_fake_evaluation_module(name, globals=None, locals=None, fromlist=(), level=0):
```

Ruff (0.16.0)

[error] Function argument `globals` is shadowing a Python builtin

(A002)